lib/pigeon/notification.ex

```diff
@@ -71,7 +71,7 @@ defmodule Pigeon.GCM.Notification do
   @moduledoc """
     Defines GCM notification struct and convenience constructor functions.
   """
-  defstruct registration_id: nil, payload: %{}, message_id: nil, updated_registration_id: nil
+  defstruct registration_id: nil, payload: %{}, message_id: nil, updated_registration_id: nil, time_to_live: nil
```

Comment:
Do we really need this? `"time_to_live"` seems to be updated within `payload`.

Reply:
Of course, yes